components/selection/selection-summary.js

```diff
@@ -39,7 +39,7 @@ class Summary extends LocalizeCoreElement(SelectionObserverMixin(LitElement)) {
 			return nothing;
 		}
 		return html`
-			<p class="d2l-body-compact">${this._summary}</p>
+			<p class="d2l-body-compact" role="status" aria-atomic="true">${this._summary}</p>
```

Comment:
Does this result in new announcements when the user changes the selection of items/rows? Or when the user interacts with Select-All? If so, are those wanted/unwanted announcements? 

Also, I am guessing Load More paging causes an announcement, which is perhaps a good thing, but does this summary announcement conflict with the focus announcement when we apply focus to the newly loaded items?